biz.aQute.bnd.gradle/src/aQute/bnd/gradle/Bndrun.groovy

```diff
@@ -144,7 +145,21 @@ public class Bndrun extends DefaultTask {
         throw new GradleException("${run.getPropertiesFile()} workspace errors")
       }
 
-      worker(run)
+      def parent = run.parent;
+      try {
+        if (parent instanceof Processor) {
+          Properties gradleProperties = new PropertiesWrapper()
+          gradleProperties.put('task', this)
+          gradleProperties.put('project', project)
+          Processor processor = new ProcessorWrapper(gradleProperties)
```

Comment:
Why do you need ProcessorWrapper? Isn't `new Processor(gradleProperties, false)` enough?